### PR TITLE
Change cache key for getIdentityFlagsFromApi

### PR DESCRIPTION
**Pull request by tm1000**
getIdentityFlagsFromApi should not be using the Global cache key prefix

### CODE REVIEW

File: src/Flagsmith.php [hunk 1]
```diff
@@ -419,7 +419,7 @@ private function getIdentityFlagsFromApi(string $identifier, ?object $traits): F
         try {
             $data = IdentitiesGenerator::generateIdentitiesData($identifier, $traits);
             $apiFlags = $this->cachedCall(
-                'Global',
+                'Identity.'.$identifier,
```
Comment (matthewelwell) [inline src/Flagsmith.php:422]: The cache key here should also include a hash of the traits for that user if provided.

**Review by matthewelwell — CHANGES_REQUESTED** [review]

**Comment by tm1000** [comment]
@matthewelwell can this be merged?

**Review by matthewelwell — APPROVED** [review]

**Comment by tm1000** [comment]
@dabeeeenster @matthewelwell would you mind issuing a release

**Comment by dabeeeenster** [comment]
Done!

**Pull request merged**